ctyparser/bigcty.py

```diff
@@ -92,32 +114,56 @@ def import_dat(self, dat_file: Union[str, os.PathLike]) -> None:
         with dat_file.open("r") as file:
             cty_dict = dict()
 
+            # get the version from the file, set filepointer to beginning of file afterwards
             ver_match = re.search(self.regex_version_entry, file.read())
             self._version = ver_match.group(1) if ver_match is not None else ""
             file.seek(0)
 
+            # variable to store last DXCC introduction with countryname, zones, etc. for further use
             last = ''
+            
             while True:
+                # read a line
                 line = file.readline().rstrip('\x0D').strip(':')
                 if not line:
                     break
+                # check if the line introduces new DXCC (line is starting with character)
+                # samples:
+                # Switzerland:              14:  28:  EU:   46.87:    -8.12:    -1.0:  HB:
+                # Sicily:                   15:  28:  EU:   37.50:   -14.00:    -1.0:  *IT9:
                 if line != '' and line[0].isalpha():
+                    # split line by ':' and remove spaces
                     segments = [x.strip() for x in line.split(':')]
+                    # check if last segment starts with '*'
+                    # A “*” preceding this prefix indicates that the country is on the DARC WAEDC 
+                    # list, and counts in CQ-sponsored contests, but not ARRL-sponsored contests
                     if segments[7][0] == '*':
+                        # remove '*' and add a marker to country name
                         segments[7] = segments[7][1:]
                         segments[0] += ' (not DXCC)'
```

Comment:
again, comment why not what

```suggestion
                    # check if this entity is not a DXCC
                    if segments[7][0] == '*':
                        segments[7] = segments[7][1:]
                        segments[0] += ' (not DXCC)'
```